sdk/openai/azure-ai-openai-assistants/src/test/java/com/azure/ai/openai/assistants/AzureVectorStoreSyncTests.java

```diff
@@ -102,11 +112,48 @@ public void listVectorStore(HttpClient httpClient, AssistantsServiceVersion serv
     @MethodSource("com.azure.ai.openai.assistants.TestUtils#getTestParameters")
     public void createVectorStoreFile(HttpClient httpClient, AssistantsServiceVersion serviceVersion) {
         beforeTest(httpClient, serviceVersion);
+        VectorStoreFile vectorStoreFile = client.createVectorStoreFile(vectorStore.getId(), fileIds.get(0));
+        assertVectorStoreFile(vectorStoreFile);
+    }
 
-        String storeId = vectorStore.getId();
-        VectorStoreFile vectorStoreFile = client.createVectorStoreFile(storeId, fileIds.get(0));
-        assertNotNull(vectorStoreFile);
-        assertNotNull(vectorStoreFile.getId());
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("com.azure.ai.openai.assistants.TestUtils#getTestParameters")
+    public void createVectorStoreFileWithAutoChunkingStrategy(HttpClient httpClient, AssistantsServiceVersion serviceVersion) {
+        beforeTest(httpClient, serviceVersion);
```

Comment:
the addition of `beforeTest` is not part of this PR but I think it's really nice. Have you tried moving into `VectorStoreTestBase` and annotating it with `@Before` from JUnit? It might save a couple of lines here and there.

Reply:
(1) BeforeEach doesn't take any method signature. (Method 'beforeTest' annotated with '@BeforeEach' should not declare parameter 'httpClient' )
(1) The client could be either of SyncClient or AsyncClient